### DISCUSSION TITLE
Não consigo me cadastrar para os benefícios estudantis

### DISCUSSION THREAD

**Discussion by YasminViktoria** [community/community#21097] [GitHub Education]
Sempre que tento me cadastrar, pede para enviar algo que comprove meu vínculo com a instituição. Eu tento enviar meu comprovante de matrícula, mas simplesmente não vai, fiquei esperando e nada, eu aperto para processar dados e nada acontece

**Comment by thuongtruong1009** [comment]
Hi @YasminViktoria 
To sign up for GitHub Education, you must be a student who has your school's student ID and you have registered your Github account with your education mail account
Then you have to fill in all the information that Github asks for
Finally, you wait about 1-2 months for Github to verify your information, and you will enjoy all the benefits when you have a Github Education account
